app/jinja_filters.py

```diff
@@ -43,6 +43,18 @@ def format_date(value):
     return value.strftime('%-d %B %Y')
 
 
+@blueprint.app_template_filter()
+def format_conditional_date(dates):
+    """ This function format_cond_date accepts two dates, if a date is submitted that is used and is available,
```

Comment:
When documenting the arguments and return values in the docstring we should follow the standard used elsewhere e.g. https://github.com/ONSdigital/eq-survey-runner/blob/master/app/data_model/answer_store.py#L153